bentoctl/cli/__init__.py

```diff
@@ -142,36 +142,26 @@ def build(
 
     deployment_config = DeploymentConfig.from_file(deployment_config_file)
     deployment_config.set_bento(bento_tag)
-    local_docker_tag = deployment_config.generate_local_image_tag()
+    local_tag = deployment_config.generate_local_image_tag()
 
     generate_deployable_container(
-        tag=local_docker_tag,
+        tag=local_tag,
         deployment_config=deployment_config,
         cleanup=get_debug_mode(),
     )
 
     if not dry_run:
-        (
-            repository_url,
-            username,
-            password,
-        ) = deployment_config.create_repository()
-
+        repository_url, _, _ = deployment_config.create_repository()
         console.print(f"Created the repository {deployment_config.repository_name}")
-        repository_image_tag = deployment_config.generate_docker_image_tag(
-            repository_url
-        )
-        tag_docker_image(local_docker_tag, repository_image_tag)
-        push_docker_image_to_repository(
-            repository=repository_image_tag,
-            username=username,
-            password=password,
```

Comment:
User can push to ECR with username/password, without logging into with docker CLI locally. With the new implementation, does the user have to login first?

Reply:
oh right, let me add that